src/dali/plugin/input/csv/blockfi.py

```diff
@@ -63,11 +63,17 @@ def __init__(
         account_holder: str,
         transaction_csv_file: str,
         trade_csv_file: Optional[str] = None,
+        addresses: Optional[str] = None,
```

Comment:
This doesn't seem the right model: many addresses could point to the same account.